### DISCUSSION TITLE
Issues when sending torrent file to qbt

### DISCUSSION THREAD

**Discussion by mshorey** [cross-seed/cross-seed#334] [Q&A] [answered]
I have cross-seed set to daemon mode.  I have set the curl -XPOST http://localhost:2468/api/webhook --data-urlencode "name=%N" in qbittorrent.  When a torrent finishes from my main tracker and it sends that command to cross-seed I get the following error:

info: [server] Received search request: { name: 'New.Amsterdam.2018.S05E05.720p.HDTV.x264-SYNCOPY' }
error: could not find a torrent with the criteria { name: 'New.Amsterdam.2018.S05E05.720p.HDTV.x264-SYNCOPY' }

The torrent does exist on the tracker I'm trying to cross seed to.  I can find it by that name on their search tool on their website and I can find it by that name in prowlarr.

If I have cross-seed set to 'search' things work as expected:
Configuration file already exists.
info: Validating your configuration...
info: Your configuration is valid!
info: [search] Found 12 torrents, 1 suitable to search for matches
info: [1/1] Searching for New.Amsterdam.2018.S05E05.720p.HDTV.x264-SYNCOPY
info: Found New.Amsterdam.2018.S05E05.720p.HDTV.x264-SYNCOPY on RevolutionTT - saved
info: [search] Found 1 cross seeds from 1 original torrents

But I'd really like to have it work in daemon mode so whenever a torrent finishes on qbt it sends the job to cross-seed>prowlarr>qbittorrent to start the download and seed.  Maybe I'm missing something pretty dumb?  

I've also tried just setting cadence to search every 10 minutes which I'd be fine with as well and it doesn't seem to search at all after setting that.

Thanks btw for a great tool!  I feel like I'm really close to getting it working like I want it to but not quite there yet.

**Comment by zakkarry** [comment, marked as answer]
I'd recommend you switch to using torrent hash, and ensure that your torrentDir is set to BT_Backup.

change your `name=%N` to `infoHash=%I` and try that.
